dataset/classification_datasets/graph_dataset.py

```diff
@@ -56,10 +56,14 @@ def process(self):
             e = json.loads(graph["edges"])
             v = json.loads(graph["vertexes"])
 
-            x = torch.LongTensor([self.v_type2id[v_["label"]] for v_ in v])
```

Comment:
To avoid multiple calling for the UNKNOWN label:
```
unk_id = self.v_type2id["UNKNOWN"]
x = torch.tensor([self.v_type2id.get(v_["label"], unk_id) for v_ in v], dtype=torch.long)
```